ban/http/api.py

```diff
@@ -170,15 +172,22 @@ def get_resource(self, identifier):
 
         parameters:
             - $ref: '#/parameters/identifier'
+              name: identifier
```

Comment:
One solution is to put a static definition into schema.py :
```
    'parameters': {
        'identifier': {
            'name': 'identifier',
            'in': 'path',
            'type': 'string',
            'required': True,
            'description': 'resource identifier'
        }
```

for each ban/http/api.py properties, we just have to define :
```
        parameters:
            - $ref: '#/parameters/identifier'
```

What do you think about this?